src/utils/OWLConversor/OWLConversor.js

```diff
@@ -0,0 +1,145 @@
+import './xml2json.js';
+import './Utils.js';
+import './Query.js';
+
+(function(){
+	window.MAX_OBJECTS_PER_ROW = 5;
+	
+	var x2js = new X2JS({attributePrefix :"@"});
+	
+	var Config = require("./config.terms.json");
+	var IgnoreList = null;
+	var RemoveList = null;
+	
+	var fn = {
```

Comment:
I don't get why you declare this `fn` variable. You can create the functions on this file en use them whenever you need it.